File: src/main/java/providers/UserProvider.java
package providers;

import models.User;
import com.github.javafaker.Faker;
import io.qameta.allure.Step;

public class UserProvider {
    @Step("Random user")
    public static User getRandom() {
        Faker faker = new Faker();
        return new User(CredentialsProvider.getRandom().getEmail(), CredentialsProvider.getRandom().getPassword(), faker.name().firstName());
    }
}

File: src/main/java/providers/UserProvider.kt
package providers

import com.github.javafaker.Faker
import models.User

class UserProvider {

    companion object{
        @JvmStatic
        // @Step("Random user")
        fun getRandom(): User{
            val faker = Faker()
            return User(CredentialsProvider.getRandom().email, CredentialsProvider.getRandom().password, faker.name().firstName())
        }
    }
}
